docs/tutorial/creating_execs_and_libs.rst

```diff
@@ -122,3 +122,22 @@ the linker removing unused symbols in the larger library.
   Due to record keeping on BLT's part to make object libraries as easy to use
   as possible, you need to define object libraries before you use them
   if you need their inheritable information to be correct.
+
+If you are using separable CUDA compilation (relocatable device code) in your
+object library, users of that library will be required to use NVCC to link their
+executables - in general, only NVCC can perform the "device link" step.  To remove
+this restriction, you can enable the ``CUDA_RESOLVE_DEVICE_SYMBOLS`` property on
```

Comment:
Should they set this on non-object libraries? Or does this happen automatically for them?

Reply:
CMake enables this automatically for shared libraries (and it can be enabled for static libraries as well); I've added this to the docs here.